low-level/files/opt/seagate/sspl/conf/sspl.config.conf.tmpl

```diff
@@ -0,0 +1,45 @@
+cortx:
```

Comment:
There needs to be template for 1-node and 3-node. 
Naming to be followed. 
1. sspl.config.tmpl.1-node
2. sspl.config.tmpl.3-node
You can create copy including sample values in 
1. sspl.config.tmpl.1-node.sample
2. sspl.config.tmpl.3-node.sample.

Reply:
Done.